include/config/config_debug_controls.h

```diff
@@ -69,4 +69,295 @@
 // Frame advancing
 #define FA_CONTROL BTN_DUP
 
+// -------------------------------------------
+
+/**** [MAP SELECT CONTROLS] ****/
+// Controller port to use
+#define MAP_SELECT_CONTORLLER_PORT CONTROLLER_PORT_1
+
+// Set to true to use a button combo
+#define MAP_SELECT_BTN_COMBO true
+#define MAP_SELECT_BTN_HOLD_FOR_COMBO BTN_L | BTN_R
+
+// Opening the map select
+#define MAP_SELECT_OPEN BTN_Z
+
+// -------------------------------------------
+
+/**** [INVENTORY EDITOR CONTROLS] ****/
+// Controller port to use
+#define INV_EDITOR_CONTORLLER_PORT CONTROLLER_PORT_1
+
+// Set to true to use a button combo
+#define INV_EDITOR_BTN_COMBO false
+#define INV_EDITOR_BTN_HOLD_FOR_COMBO BTN_Z
+
+// Opening the inventory editor
+#define INV_EDITOR_OPEN BTN_L
+
+// -------------------------------------------
+
+/**** [EVENT EDITOR CONTROLS] ****/
+// Controller port to use
+#define EVENT_EDITOR_CONTORLLER_PORT CONTROLLER_PORT_1
+
+// Set to true to use a button combo
+#define EVENT_EDITOR_BTN_COMBO true
```

Comment:
imo this sounds like it would be a `BTN_` combination, not a boolean

Reply:
I need to cleanup some of names yeah, this should be like ``EVENT_EDITOR_USE_BTN_COMBO``